src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -143,41 +123,7 @@ public void assigner_has_grant_type_specific_and_permission_manage_can_assign_ot
     }
 
     @Test
-    public void assigner_has_grant_type_specific_and_permission_manage_own_should_assign_a_task_to_assignee_with_manage_own_permission() {
```

Comment:
assigner with manage, own not in IT

Reply:
Assigner only need manage permission to assign a task. It doesn't make any difference by having own permission or not. PostTaskAssignByIdControllerTest::assigner_should_assign_a_task_to_assignee_with_grant_type_specific test cover that scenario. Also CftQueryServiceAssignTaskTest::should_retrieve_a_task_when_grant_type_specific there to cover role assignment verification.